warning: declaration shadows a field of 'vkb::PhysicalDevice'

I'm getting this warning when including the VkBootstrap header

````
warning: declaration shadows a field of 'vkb::PhysicalDevice'
```

And it comes from this template
```
include/VkBootstrap.h:545:78: warning: declaration shadows a field of 'vkb::PhysicalDevice' [-Wshadow]
    template <typename T> bool enable_extension_features_if_present(T const& features) {
                                                                             ^
include/VkBootstrap.h:505:30: note: previous declaration is here
    VkPhysicalDeviceFeatures features{};
```
https://github.com/charles-lunarg/vk-bootstrap/blob/main/src/VkBootstrap.h#L545

Is it fine to fix it? Somehow I can't make clang to ignore this warning

--- a/src/VkBootstrap.h
+++ b/src/VkBootstrap.h
@@ -542,8 +542,8 @@ struct PhysicalDevice {
 
     // If the features from the provided features struct are all present, make all of the features be enable on the
     // device. Returns true all of the features are present.
-    template <typename T> bool enable_extension_features_if_present(T const& features) {
-        return enable_features_node_if_present(detail::GenericFeaturesPNextNode(features));
+    template <typename T> bool enable_extension_features_if_present(T const& features_check) {
+        return enable_features_node_if_present(detail::GenericFeaturesPNextNode(features_check));
     }
 
     // A conversion function which allows this PhysicalDevice to be used

